Header header
uint8 temp_1_curr
uint8 temp_1_min
uint8 temp_1_max
uint8 temp_2_curr
uint8 temp_2_min
uint8 temp_2_max
uint8 temp_3_curr
uint8 temp_3_min
uint8 temp_3_max
uint8 temp_4_curr
uint8 temp_4_min
uint8 temp_4_max
uint8 temp_5_curr
uint8 temp_5_min
uint8 temp_5_max
uint8 temp_6_curr
uint8 temp_6_min
uint8 temp_6_max
uint8 akku_voltage_curr
uint8 akku_voltage_min
uint8 akku_voltage_max
uint8 hals_motor_voltage_curr
uint8 hals_motor_voltage_min
uint8 hals_motor_voltage_max
uint8 hals_logik_voltage_curr
uint8 hals_logik_voltage_min
uint8 hals_logik_voltage_max
uint8 tablett_logik_voltage_curr
uint8 tablett_logik_voltage_min
uint8 tablett_logik_voltage_max
uint8 arm_logik_voltage_curr
uint8 arm_logik_voltage_min
uint8 arm_logik_voltage_max
uint8 tablett_motor_voltage_curr
uint8 tablett_motor_voltage_min
uint8 tablett_motor_voltage_max
uint8 hals_motor_current_curr
uint8 hals_motor_current_min
uint8 hals_motor_current_max
uint8 hals_logik_current_curr
uint8 hals_logik_current_min
uint8 hals_logik_current_max
uint8 tablett_logik_current_curr
uint8 tablett_logik_current_min
uint8 tablett_logik_current_max
uint8 arm_logik_current_curr
uint8 arm_logik_current_min
uint8 arm_logik_current_max
uint8 tablett_motor_current_curr
uint8 tablett_motor_current_min
uint8 tablett_motor_current_max